apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,7 +157,3 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
-
-.conversation-view .panel {
```

Comment:
I meant that you have a 
```css
.panel {
  /* ...*/
  left: 0;
  /* ... */ 
}
 ```
In `./apps/sms/views/shared/style/panel.css:1`. Dead code in the sense of a liveliness analysis, if this makes sense :-) I might have missed something though.